src/main/java/uk/gov/hmcts/reform/sendletter/api/SendLetterResponse.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sendletter.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.UUID;
+
+public class SendLetterResponse {
+
+    @JsonProperty("letter_id")
+    public final UUID letterId;
+
+    public SendLetterResponse(UUID letterId) {
```

Comment:
I think you'll need that `@JsonProperty` here

Reply:
the parameter is public so i think it will use the field, this is (basically) exact copy from letter-sender-producer

Reply:
letter-sender-producer sends this model, here you consume it, hence other annotation placement